src/backupNg/backupNg.spec.js

```
@@ -11,6 +12,84 @@ const DEFAULT_SCHEDULE = {
   cron: "0 * * * * *",
 };
 
+const validateRootTask = (log, props) =>
+  expect(log).toMatchSnapshot({
+    end: expect.any(Number),
+    id: expect.any(String),
+    jobId: expect.any(String),
+    scheduleId: expect.any(String),
+    start: expect.any(Number),
+    ...props,
+  });
+
+const validateVmTask = (task, vmId, props = {}) => {
+  if (task.status !== "success") {
+    props.result = {
+      stack: expect.any(String),
+    };
+  }
+  expect(task).toMatchSnapshot({
+    data: {
+      id: expect.any(String),
+    },
+    end: expect.any(Number),
+    id: expect.any(String),
+    message: expect.any(String),
+    start: expect.any(Number),
+    ...props,
+  });
+  expect(task.data.id).toBe(vmId);
+};
+
+const validateSnapshotTask = (task, props) =>
+  expect(task).toMatchSnapshot({
+    end: expect.any(Number),
+    id: expect.any(String),
+    result: expect.any(String),
+    start: expect.any(Number),
+    ...props,
+  });
+
+const validateExportTask = (task, isFull, type, srOrRemoteIds, props) => {
+  if (task.status !== "success") {
+    props.result = {
```

It should be passed by the test. In case if the error is not propagated.